@vates/types/src/xo.mts

```diff
@@ -281,6 +283,19 @@ export type XoSr = BaseXapiXo & {
   usage: number
 }
 
+export type XoTask = {
+  id: Branded<'task'>
+  type: 'task'
+  start: number
+  end: number
+  properties: {
+    type: string
+    name: string
+  }
+  status: Record<string, TASK_STATUS_TYPE>
+  tasks?: XoTask[]
+}
```

Comment:
IMO, you should move this type to `vates-task.mts`
then do something like: `export type XoTask = Omit<Task, ...>`